pkg/web_css/lib/src/_pkg.scss

```diff
@@ -303,6 +303,49 @@
     stroke: var(--pub-downloads-chart-frame-color);
   }
 
+  .downloads-chart-tooltip {
+    border-radius: 5px;
+    margin: 0px;
+    padding: 2px 8px;
+    box-shadow: rgba(0, 0, 0, 0.1) 0px 4px 12px;
+    background-color:var(--pub-neutral-bgColor);
+    opacity: 0.9;
```

Comment:
Do we want this?
It might make the text in the tooltip slightly harder to read...

Reply:
I kind of like it with a little bit opacity, but I see your point too....

Reply:
Made it such that it is only the background that has a small opacity, keeping the contrast in the text and squares.